### ISSUE TITLE
v1.2.9中文路径乱码

### ISSUE THREAD

**Issue by codesboy** [cmderdev/cmder#764]
`cmder v1.2.9`中文路径乱码,该怎么解决?

**Comment by Stanzilla** [comment]
English please

**Comment by MartiUK** [comment]
or someone who can translate please?

**Comment by codesboy** [comment]
ok, i hava a question about my `cmder v1.2.9`,If I enter Chinese folder,this file path will Chinese garbled.(=￣ω￣=)
<img src="http://7xlaft.com1.z0.glb.clouddn.com/jietu/1.png" alt="cmder"><br>
<img src="http://7xlaft.com1.z0.glb.clouddn.com/jietu/2.png" alt="cmder">

**Comment by esphas** [comment]
@phpgao once gave a solution to this problem in a post ([Link]()), I'm not sure whether that works because my cmder works well with chinese characters even without further configuration.

His solution:

**Add the below 4 lines to `cmder/config/aliases`:**

```
l=ls --show-control-chars 
la=ls -aF --show-control-chars 
ll=ls -alF --show-control-chars 
ls=ls --show-control-chars -F
```

**Comment by codesboy** [comment]
@esphas It doesn't work in new version

**Comment by esphas** [comment]
No idea. (BTW I seemed to have given the wrong link.. [it should be this one](https://blog.phpgao.com/cmder.html))

**Comment by Eamonnzhang** [comment]
@Stanzilla  why this issue is closed?

**Comment by jing2si** [comment]
this problem is still there。
**cmder_mini 1.3.0**

**Comment by janschulz** [comment]
Just for clarification: in my understanding, the fix in #1070 won't change the two cases reported above in the two screenshots of https://github.com/cmderdev/cmder/issues/764#issuecomment-163801058:

The fix in #1070 only parses the current directory which is displayed one line above the current prompt, but does nothing with the current typed in command nor the  problem that clink doesn't find a file. I suspect that a newer clink version fixes both "by accident".
